src/main/resources/avro/bdg.avdl

```diff
@@ -57,8 +57,55 @@ record Contig {
   union { null, string } species = null;
 }
 
+record Sequence {
+  union { null, string } bases;
+  union { null, string } qualities;
+}
+
+/**
+   The DNA fragment that is was targeted by the sequencer, resulting in
+   one or more reads.
+*/
+record Fragment {
+  /**
+     The name of this Fragment.
+  */
+  union { null, string } readName = null;
```

Comment:
I figure `readName` should be used like the `read id` field in SAM, but without the typical trailing `/1` or `/2`.  Maybe we should change the name to something simple like `id`or `name`.